### PR TITLE
Add total nb to trie migration rpc

### PR DESCRIPTION
**Pull request by cheme**
While doing trie migration on rococo and rockmine, I realize this Rpc would be way more usefull by adding the total number of items.

This PR does it.

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-each-crate
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2079187

File: utils/frame/rpc/state-trie-migration-rpc/src/lib.rs [hunk 1]
```diff
@@ -146,11 +154,14 @@ where
 
 		let hash = at.unwrap_or_else(|| self.client.info().best_hash);
 		let state = self.backend.state_at(hash).map_err(error_into_rpc_err)?;
-		let (top, child) = migration_status(&state).map_err(error_into_rpc_err)?;
+		let (top, child, total_top, total_child) =
+			migration_status(&state).map_err(error_into_rpc_err)?;
```
Comment (lexnv) [inline utils/frame/rpc/state-trie-migration-rpc/src/lib.rs:158]: nit: Would it make sense to have this function return `MigrationStatusResult` directly?

**Comment by lexnv** [comment]
DQ: Shouldn't we also update the `migration_status` from [here](https://github.com/paritytech/substrate/blob/master/frame/state-trie-migration/src/lib.rs#L1666-L1667) and [here](https://github.com/paritytech/substrate/blob/master/frame/state-trie-migration/src/lib.rs#L1725-L1727)? The CI is broken atm I guess

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-each-crate
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2082563

**Comment by cheme** [comment]
Thanks a lot @lexnv (I did not really expect a test for this rpc).

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-each-crate
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2082731

**Review by lexnv — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Comment by cheme** [comment]
bot rebase

**Comment by cheme** [comment]
bot merge

**Comment by cheme** [comment]
bot rebase

**Comment by cheme** [comment]
bot merge

**Pull request merged**